src/index.js

```diff
@@ -48,8 +50,13 @@ const paramsHtml = (params) => {
 };
 
 const postToSmtp = ({
-  from, to, subject, params, html,
+  from, to, subject, params, html, category, uniqueArgs,
 }) => {
+  const smtpApiHeader = {
+    category: [category],
+    unique_args: uniqueArgs,
```

Comment:
Depending on your version of the API or how it is sent email, this field has a different name.

Reply:
https://docs.sendgrid.com/for-developers/sending-email/unique-arguments